header-rewrite-filter/header_rewrite_config.cc

```diff
@@ -30,7 +30,7 @@ class HttpHeaderRewriteFilterConfigFactory : public NamedHttpFilterConfigFactory
     return ProtobufTypes::MessagePtr{new envoy::extensions::filters::http::HeaderRewrite()};
   }
 
-  std::string name() const override { return "sample"; }
+  std::string name() const override { return "envoy.header_rewrite"; }
```

Comment:
Nit: would be good to put this in a const in the header

Reply:
I don't have a strong opinion either way, but other extensions just inline it https://github.com/envoyproxy/envoy/blob/main/source/extensions/health_checkers/thrift/config.h#L24